src/schema/jsii2schema.ts

```diff
@@ -525,214 +526,3 @@ function methodSchema(method: jsiiReflect.Callable, ctx: SchemaContext) {
     return basicSchema;
   });
 }
-
-export function isDataType(
```

Comment:
btw, this function is not being used anywhere. I wonder if we should use it here: https://github.com/cdklabs/decdk/blob/a1a2785c838b093322852c5ed9526a629a2e7498/src/type-resolution/resolve.ts#L168

Reply:
Maybe. 🤔 Worth a try!